CWE/events/italy.txt

```diff
@@ -44,6 +79,12 @@ country_event = {
 	option = {
 		name = EVT_8007704_B
 		define_general = { name = "Umberto II" personality = energetic background = no_background }
+		set_country_flag = umberto_II
+		any_pop = { 
+			    limit = { OR = { ideology = communist } OR = { ideology = socialist } }
```

Comment:
"OR = { ideology = communist } OR = { ideology = socialist }" is wrong, it should be "OR = { pop_majority_ideology = communist pop_majority_ideology = communist_social }". 

It is better to use scaled militancy and consciousness for these kinds of effects.
any_pop = {
		scaled_militancy = {
			factor = 5
			ideology = communist
		}
		scaled_militancy = {
			factor = 5
			ideology = communist_social
		}
		scaled_consciousness = {
				ideology = communist
				factor = 6
		}
		scaled_consciousness = {
				ideology = communist_social
				factor = 6
		}
}